src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj

```diff
@@ -21,7 +21,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
-    <Version>1.0.1-preview</Version>
+    <Version>1.1.0-preview</Version>
```

Comment:
Did we miss this in the previous PR?

Reply:
Yes, because this file have both Version and VersionPrefix...
It looks like some projects use Version, some use VersionPrefix, some use both... I will send a separate PR to unify all.

Reply:
I will do VersionPrefix => Version in a separated PR. An internal task has been created.